src/marshmallow/fields.py

```diff
@@ -1114,15 +1114,62 @@ def _make_object_from_format(value, data_format):
         return dt.datetime.strptime(value, data_format)
 
 
-class LocalDateTime(DateTime):
-    """A formatted datetime string in localized time, relative to UTC.
+class NaiveDateTime(DateTime):
+    """A formatted naive datetime string.
+
+    :param str format: See :class:`DateTime`.
+    :param timezone timezone: Used on deserialization. If None, aware datetimes
+        are rejected. If not None, aware datetimes are converted to this
+        timezone before their timezone information is removed.
+    :param kwargs: The same keyword arguments that :class:`Field` receives.
+    """
+
+    AWARENESS = "naive"
 
-        ex. ``"Sun, 10 Nov 2013 08:23:45 -0600"``
+    def __init__(self, format=None, timezone=None, **kwargs):
+        super().__init__(format=format, **kwargs)
+        self._timezone = timezone
 
-    Takes the same arguments as :class:`DateTime <marshmallow.fields.DateTime>`.
+    def _deserialize(self, value, attr, data, **kwargs):
+        ret = super()._deserialize(value, attr, data, **kwargs)
+        if is_aware(ret):
+            if self._timezone is None:
+                raise self.fail(
+                    "invalid awareness",
+                    awareness=self.AWARENESS,
+                    obj_type=self.OBJ_TYPE,
+                )
+            ret = ret.astimezone(self._timezone).replace(tzinfo=None)
+        return ret
+
+
+class AwareDateTime(DateTime):
+    """A formatted aware datetime string.
+
+    :param str format: See :class:`DateTime`.
+    :param timezone default_timezone: Used on deserialization. If None, naive
+        datetimes are rejected. If not None, naive datetimes are set this
+        timezone.
+    :param kwargs: The same keyword arguments that :class:`Field` receives.
     """
 
-    localtime = True
+    AWARENESS = "aware"
+
+    def __init__(self, format=None, default_timezone=None, **kwargs):
+        super().__init__(format=format, **kwargs)
+        self._default_timezone = default_timezone
+
+    def _deserialize(self, value, attr, data, **kwargs):
+        ret = super()._deserialize(value, attr, data, **kwargs)
+        if not is_aware(ret):
+            if self._default_timezone is None:
+                raise self.fail(
```

Comment:
`raise self.fail` doesn't actually do anything different from `self.fail`. I noticed we do this in other places and made a fix here: #1301